urllib3/util/ssl_.py

```diff
@@ -11,7 +11,6 @@
 
 SSLContext = None
 HAS_SNI = False
-create_default_context = None
```

Comment:
@sigmavirus24 because it is not longer used?

do you mean this useless var should be kept for backward compatibility?